### ISSUE TITLE
GPU training giving 0 mAP

### ISSUE THREAD

**Issue by nav232** [ultralytics/yolov5#11220]
i have trained my yolov5s custom model on CPU till now and got good accuracy, precision and recall. Now i have started training the same model with updated custom dataset on nvidia T1000 GPU but even after running for more than 100 epochs, i still get 0 precision, 0 recall and 0 mAP. Basically every time i train it with GPU i get 0 for every parameter whereas on CPU, model is training fine and giving good results. I need some help to figure out what is the problem here and how it can be resolved?

**Comment by glenn-jocher** [comment]
@nav232 it's great to hear that your YOLOv5 model is performing well on CPU! When transitioning to GPU, a common issue causing 0 mAP, precision, and recall is a discrepancy between the training and validation set paths. Ensure that the paths in your data.yaml file point to the correct directories for both sets. Additionally, ensure your GPU setup and PyTorch installation are proper. More details on this can be found in the Ultralytics Docs. If the issue persists, feel free to share more details in your training command and data.yaml file for further assistance. Keep up the great work!
